src/test/java/seedu/address/logic/commands/UnlockCommandTest.java

```diff
@@ -51,4 +51,18 @@ public void execute_unlockWrongPassword_throwsCommandException() throws NoSuchPa
         UnlockCommand command = new UnlockCommand(INCORRECT_PASSWORD);
         assertCommandFailure(command, model, UnlockCommand.MESSAGE_INCORRECT_PASSWORD);
     }
+
+    @Test
+    public void execute_equals_success() {
```

Comment:
Do you think it would be more appropriate to name the test to describe that you are testing reapeated unlocks/mutability rather than just "equals"?

Reply:
I believe the test name is correct because I'm testing the equals method to be successful when two `UnlockCommand` with the same `currentPassword` fields are tested to be equal.